src/assets/wise5/authoringTool/peer-grouping/select-peer-grouping-option/select-peer-grouping-option.component.ts

```diff
@@ -37,6 +43,8 @@ export class SelectPeerGroupingOptionComponent implements OnInit {
   setPeerGroupingLogicName(): void {
```

Comment:
Change to private since this func is only called in this class?
```suggestion
  private setPeerGroupingLogicName(): void {
```